diff --git a/cmake/FindQuaZip.cmake b/cmake/FindQuaZip.cmake
index d733e5978700..79d6c15a72f1 100644
--- a/cmake/FindQuaZip.cmake
+++ b/cmake/FindQuaZip.cmake
@@ -17,12 +17,12 @@ else()
     if(NOT QUAZIP_FOUND)
         # Try to find QuaZip version 0.x
         find_library(QUAZIP_LIBRARIES
-            NAMES quazip5 quazip
+            NAMES quazip5 quazip quazip1-qt5
             PATHS /usr/lib /usr/lib64 /usr/local/lib
         )
         find_path(QUAZIP_INCLUDE_DIRS quazip.h
             PATHS /usr/include /usr/local/include
-            PATH_SUFFIXES quazip5 quazip
+            PATH_SUFFIXES quazip5 quazip QuaZip-Qt5-1.0/quazip QuaZip-Qt5-1.1/quazip
         )
     endif()
 endif()
